deps.edn

```diff
@@ -203,8 +203,8 @@
     cloverage/cloverage          {:mvn/version "1.2.4"}
     com.gfredericks/test.chuck   {:mvn/version "0.2.13"}                    ; generating strings from regexes (useful with malli)
     djblue/portal                {:mvn/version "0.35.0"}                    ; ui for inspecting values
-    eftest/eftest                {:mvn/version "0.6.0"}
     io.github.camsaul/humane-are {:mvn/version "1.0.2"}
+    io.github.metabase/hawk      {:sha "126daf1c0aca962b9ca3a847716e0ad8e7821580"}
```

Comment:
What's stopping us from publishing it to clojars? I could help set it up if that's the reason.

Reply:
I don't really think it's worth the effort since if you're using Leiningen you can just use Eftest directly